Yoast/javascript
# Style update wizard radiobuttons and inputfields

## Summary


This PR can be summarized in the following changelog entry:

* [@yoast/components] Updates the configuration wizard to match the new design

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and the [corresponding PR on the plugin](https://github.com/Yoast/wordpress-seo/pull/15388).
* Open the configuration wizard
* Check the different steps to see if they match the new design

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * Configuration wizard

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/FRO-24
